pkg/kudoctl/cmd/package_verify.go

```diff
@@ -37,23 +40,44 @@ func newPackageVerifyCmd(fs afero.Fs, out io.Writer) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP((*string)(&verifyCmd.output), "output", "o", "", "Output format for command results.")
-
+	cmd.Flags().StringSliceVar(&verifyCmd.paramChecks, "param-checks", []string{}, "Additional parameter checks: [display, hint, desc, type, groups]")
```

Comment:
We should probably have `all` to simplify just checking everything.

Reply:
Good point. Haven't used that because I have to fix so much in Cassandra 🙈